haskell/repl.bzl

```diff
@@ -483,8 +483,8 @@ def _create_repl(hs, cc, posix, ctx, repl_info, output):
         is_executable = True,
         substitutions = {
             "%{ENV}": render_env(env),
-            "%{TOOL}": hs.tools.ghci.path,
-            "%{ARGS}": "(" + " ".join(
+            "%{TOOL}": hs.tools.ghc.path,
+            "%{ARGS}": "--interactive (" + " ".join(
```

Comment:
Does this work for other GHC versions as well? If so, great! If not, we'll have to gate this to not break the other GHC versions.

Reply:
Good question. To investigate.